commit 26aef0a236e7053ab3cf9127962b1d50d6a6dc4c
Author: Leo Gao <54557097+leogao2@users.noreply.github.com>
Date:   Fri Nov 13 11:45:27 2020 -0700

    Disable threading in lmd
    
    This fixes the "daemonic processes are not allowed to have children" error

diff --git a/data/create_tfrecords.py b/data/create_tfrecords.py
index 500fdb1..7ede09f 100644
--- a/data/create_tfrecords.py
+++ b/data/create_tfrecords.py
@@ -121,7 +121,7 @@ class EncodedCompressedReader:
     def __init__(self, f, encoder, separator=None, fix=False, minimum_size=0):
         def _gen():
             # Generator yielding the files inside the archive as encoded streams
-            g = Reader(f).stream_data()
+            g = Reader(f).stream_data(threaded=False)
             for s in g:
                 yield BufferedEncodedStream(s, encoder, separator, fix, minimum_size, text_mode=True)
 
